### PR TITLE
Namespace for Jaeger secret creation

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

**Review by kminder — APPROVED** [review]
Apprvoed.

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 1]
```diff
@@ -96,8 +96,25 @@ deployment.apps/verrazzano-prod-query       1/1     1            1           52m
 
 ## Configure an application to export traces to Jaeger
 
-If your application is configured to use tracing libraries, or in the Istio mesh, you can instruct Jaeger to export those traces using annotations.
-To export traces, annotate your applications with the `"sidecar.jaegertracing.io/injected": "true"` annotation.
+The Jaeger agent sidecar is injected to application pods by 
+`"sidecar.jaegertracing.io/inject": "true"` annotation. You may apply this annotation to namespaces, or pod controllers such as Deployments.
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:100]: by `"sidecar.jaegertracing.io/inject": "true"` annotation. -> by the `"sidecar.jaegertracing.io/inject": "true"` annotation.
namespaces, or pod -> namespaces or pod (no comma needed)

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

**Review by bhatpmk — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]
Fine.

**Pull request merged**